app/controllers/carts_controller.rb

```diff
@@ -1,12 +1,15 @@
 class CartsController < ApplicationController
   def create
     update_cart
-    flash[:notice] = "Added to Cart"
-    redirect_to projects_path
+    flash[:notice] = 'Added to Pending Loans'
+    redirect_to pending_loan_path
```

Comment:
Since I missed out on your Turing party today, can you give some context for this switch from Cart to Pending Loans?